tests/test_remove_mon_from_cluster.py

```diff
@@ -0,0 +1,126 @@
+"""
+A Testcase to remove one or more mons
+when I/O's are happening.
+
+Polarion-ID- OCS-355
+
+"""
+
+import logging
+import re
+import random
+from ocs import ocp, constants
+from resources import pod
+from ocsci.testlib import tier4, ManageTest
+from ocsci.config import ENV_DATA
+from ocs.exceptions import CephHealthException
+from utility.utils import run_cmd
+
+
+log = logging.getLogger(__name__)
+
+DEP = ocp.OCP(
+    kind=constants.DEPLOYMENT, namespace=ENV_DATA['cluster_namespace']
+)
+POD = ocp.OCP(
+    kind=constants.POD, namespace=ENV_DATA['cluster_namespace']
+)
+
+
+def get_mons_from_cluster():
+    """
+    Getting the list of active mons in the cluster
+    if available = 1 then pod is active or else its inactive/
+    does not exist.
+    """
+
+    ret = DEP.get(resource_name='', out_yaml_format=False,
+                  selector='app=rook-ceph-mon')
+    available_mon = re.findall(r'[\w-]+mon-+[\w-]', ret)
+    return available_mon
+
+
+def verify_mons_are_present():
+    """
+    To verify if there are more than 1 mon
+    present in the cluster
+    """
+    return bool(len(get_mons_from_cluster()))
+
+
+def remove_mon_from_cluster(mon):
+    """
+    Removing the mon pod from deployment
+    """
+    assert DEP.delete(resource_name=mon)
+
+
+def verify_mon_pod_up():
+    ret = POD.wait_for_resource(
+        condition=constants.STATUS_RUNNING, selector='app=rook-ceph-mon',
+        resource_count=5, timeout=700)
+    return ret
+
+
+def run_io():
+    """
+    Run io on the mount point
+    """
+
+    cmd = 'ceph osd lspools'
+    tools_pod = pod.get_ceph_tools_pod()
+    out = tools_pod.exec_ceph_cmd(ceph_cmd=cmd)
+    cmd1 = ('rados bench -p %s 60 write -t 32 -b 409600000 --no-cleanup'
```

Comment:
This is an interesting implementation of IO execution. Please try to modify helpers run IO function with that

Reply:
Added this function in resources/pod.py